src/js/components/Select/__tests__/Select-test.js

```diff
@@ -1709,6 +1715,208 @@ describe('Select', () => {
     expect(getAllByRole('option')[0]).toHaveFocus();
   });
 
+  test('allows keyboard navigation with search and top clear', async () => {
+    // Neither user.type() nor user.keyboard() trigger arrow keys as needed
```

Comment:
Specifically, I noticed events generated from either would lack both `keyCode` and `which`, which the Keyboard implementation relies on for routing callbacks.